pkgs/servers/nats-server/2.6.0-CVE-2022-26652.patch

```diff
@@ -0,0 +1,40 @@
+Based on upstream https://github.com/nats-io/nats-server/commit/b4128693ed61aa0c32179af07677bcf1d8301dcd
+with test changes removed (as we don't run them (yet)) and
+the path -> filepath changes omitted as it is for the benefit
+of windows, which we don't really support
+
+--- a/server/stream.go
++++ b/server/stream.go
+@@ -3620,6 +3619,17 @@
+ 	}
+ 	defer os.RemoveAll(sdir)
+ 
++	logAndReturnError := func() error {
++		a.mu.RLock()
++		err := fmt.Errorf("unexpected content (account=%s)", a.Name)
++		if a.srv != nil {
++			a.srv.Errorf("Stream restore failed due to %v", err)
++		}
++		a.mu.RUnlock()
++		return err
++	}
++	sdirCheck := filepath.Clean(sdir) + string(os.PathSeparator)
```

Comment:
```suggestion
+	sdirCheck := path.Clean(sdir) + string(os.PathSeparator)
```

Reply:
Well in these cases it definitely is a filepath, and really I'd prefer sticking with the new clause verbatim.